src/pymatgen/io/vasp/sets/lobster.py

```diff
@@ -0,0 +1,107 @@
+# ruff: noqa: PGH003
```

Comment:
Thanks a lot for the wonderful work, I totally agree inputs/outputs VASP is too large to navigate through. 

Not sure if this is planned to be fixed or not, if not can you please add a TODO tag and pin me to fix these `PGH003` violations (as I cannot push commits directly) ? Also I believe it's a pretty helpful rule to have (ignore only specific rules instead everything). 

Thanks for the work again!